test/plugin_functional/test_suites/core_plugins/applications.ts

```diff
@@ -99,12 +105,20 @@ export default function({ getService, getPageObjects }: PluginFunctionalProvider
       await PageObjects.common.navigateToApp('chromeless');
       await loadingScreenNotShown();
       expect(await testSubjects.exists('headerGlobalNav')).to.be(false);
+
+      const wrapperWidth = await getAppWrapperWidth();
+      const windowWidth = (await browser.getWindowSize()).width;
+      expect(wrapperWidth).to.eql(windowWidth);
```

Comment:
I haven't tried yet, but probably it's safer to test with `Percy` in `visual_regression` setup.

Reply:
I just tried it, however it seems it needs a specific configuration and/or can't be properly used from the `plugin_functional` test suite, or at least I did not achieve to (the snapshot fails). As the test is pretty simple and only checks a dom element's width, I think it's acceptable for now.